examples/navier-stokes/nsplex.c

```diff
@@ -1028,10 +1034,35 @@ int main(int argc, char **argv) {
   user->dt = dt;
 
   // Solve
+  start = clock();
   ierr = TSSolve(ts, Q); CHKERRQ(ierr);
+  ierr = TSGetSolveTime(ts,&ftime); CHKERRQ(ierr);
+  end = clock();
+  cpu_time_used = ((double) (end - start)) / CLOCKS_PER_SEC;
+  ierr = PetscPrintf(PETSC_COMM_WORLD,
+                    "Time taken for solution: %g\n",
+                    (double)cpu_time_used); CHKERRQ(ierr);
```

Comment:
If you want to time the solution, I'd use lines `267-272` of `bps.c` in the `examples/petsc` folder.

Reply:
Fixed in 043888d